header.php

```diff
@@ -41,13 +41,13 @@
     <link rel="shortcut icon" type="image/png" href="img/favicon/favicon-32x32.png" sizes="32x32">
 
     <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+//     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+//     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+//     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+//     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-    ga('create', 'UA-40768937-6', 'primecoin.io');
-    ga('send', 'pageview');
+//     ga('create', 'UA-40768937-6', 'primecoin.io');
+//     ga('send', 'pageview');
 
```

Comment:
Why comment out google analytics?

Reply:
I was hosting it elsewhere, but I've re-enabled that for this PR now